plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -316,8 +316,8 @@ void GSSettingsDlg::UpdateRenderers()
 	}
 	else
 	{
-		//GSRendererType best_renderer = (level >= D3D_FEATURE_LEVEL_10_0) ? GSRendererType::DX1011_HW : GSRendererType::DX9_HW;
-		renderer_setting = static_cast<GSRendererType>(theApp.GetConfigI("Renderer"));
+		GSRendererType INI_Renderer = GSRendererType(theApp.GetConfigI("Renderer"));
+		renderer_setting = (INI_Renderer == GSRendererType::Undefined) ? GSUtil::GetBestRenderer() : INI_Renderer;
```

Comment:
`INI_Renderer` naming is inconsistent with surrounding code. (Optionally, the variable isn't needed if you adapt my previous comment).